### PR TITLE
Separate profile page to a new screen.

### LINKED ISSUE #4: Create a standalone profile page
a new profile page is needed: Separate out the current tests. Have username Bio real name * DOB
Make this separate from the main page which shows as it is today with verifications, username and bio (but not editable) along with the add verification button.

### PR DESCRIPTION
**Pull request by wparad**
Fixes #4

### CODE REVIEW

**Comment by HenryHins** [comment]
Do you think we should make a sepearte save button for the public profile information? Some users may not want to provide us their private information once after they registered

**Comment by wparad** [comment]
> Some users may not want to provide us their private information once after they registered.

We could, but if they don't provide that information they won't be able to have their tests verified.  I move the profile fill information from the release form.  Now it gets populated from there?

Is that what you mean?

**Comment by HenryHins** [comment]
I guess it looks good for now. There aren't enought fields for two seperate save buttons anyway.

File: content/view/view.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h4 id="publicProfileUsername">{{username}}</h4>
 		<hr>
 	</div>
 	<div ng-show="!linkname && !error" class="do-not-animate">
-			<h4 id="publicProfileUsername">Retrieving Profile...</h4>
+			<img id="spinner" ng-src="image/spinner.gif" class="fadeFast">
```
Comment (HenryHins) [inline content/view/view.html]: Won't show the spinner properly here but I will make a PR to fix it after you've merged this one.

**Review by HenryHins — APPROVED** [review]

**Comment by wparad** [comment]
I noticed that, what did I miss?

Edit: I see, the css tag is putting it too low in the stack.  I wonder if we shouldn't have a css file per component?

**Comment by wparad** [comment]
Also I'm planning on switching the order of the data, Medical information up top with the secondary information on the bottom.

**Pull request merged**